### ISSUE TITLE
Introductory song transition playing even when disabled

### ISSUE THREAD

**Issue by JerseyJimmy** [hmxmilohax/rock-band-3-deluxe#802]
**Describe the bug**
Even if song transitions are set to [None], the one preceding a setlist will always play no matter what.

**To Reproduce**
1. Go to Options > Deluxe Settings > Venue Options >Menu Backgrounds... > Song Transitions
2. Select [None]
3. Play any song or setlist in Quickplay

**Expected behavior**
The song transition prior to the setlist should be skipped, meaning there is simply a loading screen before immediately commencing the song.

**DX Settings**
- `dx_settings.dta`: [dx_settings.txt](https://github.com/user-attachments/files/16401954/dx_settings.txt)
- `dx_modifiers.dta`: [dx_modifiers.txt](https://github.com/user-attachments/files/16401955/dx_modifiers.txt)
- `dx_values.dta`: [dx_values.txt](https://github.com/user-attachments/files/16401957/dx_values.txt)

**Game Information**
 - Platform: RPCS3
 - Version: Latest nightly
 - Song Count: 3478

**Comment by jnackmclain** [comment]
Yeah things got kind of left in this state currently due to some issues and I never went back to fix it. I need to desperately

**Comment by jnackmclain** [comment]
fixed in latest nightly
